addon/serializer.js

```diff
@@ -8,7 +8,7 @@ export default JSONAPISerializer.extend({
     return underscore(attr);
   },
 
-  keyForRelationship: function(rawKey, kind) {
+  keyForRelationship: function(rawKey) {
```

Comment:
does changing it to: `keyForRelationship: function(rawKey /*, kind */) {` works for you?

IMHO, this might work better as a todo option.